### PR TITLE
refactor(TJS): update favicons and refactor mobile menu

### PR DESCRIPTION
**Pull request by DrShpongle**
This updates favicons (provides full set including MS tiles, mobile safari etc).
Also it fixes the empty mobile menu shown for non logged in users.

<details>
  <summary>Before:</summary>
  
  
</details>
<details>
  <summary>After:</summary>
  
  
</details>

Driving Car Wash GIF by BlackBearDiner

### CODE REVIEW

File: apps/testing-javascript/.env.production [hunk 1]
```diff
@@ -5,8 +5,7 @@ NEXT_PUBLIC_APP_NAME=testing-javascript
 INNGEST_APP_NAME=testing-javascript
 NEXT_PUBLIC_SITE_TITLE="Testing JavaScript"
 NEXT_PUBLIC_HOST=testingjavascript.com
-# NEXT_PUBLIC_URL=https://testingjavascript.com
-NEXT_PUBLIC_URL=https://testing-javascript.vercel.app
+NEXT_PUBLIC_URL=https://testingjavascript.com
```
Comment (DrShpongle) [inline apps/testing-javascript/.env.production:8]: @jbranchaud I guess it should be fixed in this way?

File: apps/testing-javascript/src/components/navigation.tsx [hunk 2]
```diff
@@ -243,45 +243,39 @@ const MobileNav = () => {
...
-            <NavigationMenu.List className="flex h-full items-center justify-center">
-              <NavigationMenu.Item className="h-full">
-                <NavigationMenu.Trigger className="flex h-full items-center justify-center rounded-full px-2 py-2 hover:bg-gray-100">
-                  <MenuIcon className="h-5 w-5" aria-hidden />
-                  <span className="sr-only">Menu</span>
-                </NavigationMenu.Trigger>
-                <NavigationMenu.Content className="absolute left-0 top-full w-full border-t border-gray-100 bg-white shadow-2xl shadow-gray-500/20">
-                  <ul>
-                    {/* <MobileNavLink ... /> */}
-                    {status === 'authenticated' && (
```
Comment (DrShpongle) [inline apps/testing-javascript/src/components/navigation.tsx:257]: all content in this menu should be visible for looged in users only so I moved this condition on the top level

File: apps/testing-javascript/.env.production (hunk 1 above)
Comment (jbranchaud) [reply apps/testing-javascript/.env.production:8]: I believe it is overridden by vercel env vars, but still a good thing to update. 🙌

**Review by jbranchaud — APPROVED** [review]
fantastic, thanks for catching all of this!

File: apps/testing-javascript/.env.production (hunk 1 above)
Comment (DrShpongle) [reply apps/testing-javascript/.env.production:8]: not sure, there is no such env at TJS vercel:
CleanShot 2024-01-10 at 16 19 36@2x

**Pull request merged**